### ISSUE TITLE
Constant condition in comprehension is not reported

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#1864]
Reproduction:

```python
comp = [num for num in range(5) if True]
```

**Comment by sobolevn** [comment]
The same happens for:

```python
[
    ...
    for x in ...
    if (some if some > 1 else other)
]
```

It should raise `NestedTernaryViolation`.
